README.md

```
@@ -0,0 +1,256 @@
+# 🎬 모두의 플리 (MOPL)
+
+<div align="center">
+
+### 대규모 트래픽을 고려한 글로벌 콘텐츠 큐레이션 & 소셜 플랫폼
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)